### ISSUE TITLE
Tag new release

### ISSUE THREAD

**Issue by llvilanova** [aclements/libelfin#53]
Please tag current master for a new release, so latest fixes can be included in Debian.

**Comment by petterreinholdtsen** [comment]
Yes, please do.  I am one of the Debian maintainers and would love to see tagged versioned releases.  Or if the project is no longer developed, declare this project abandoned so others can consider taking over.
